src/workspacer.FSharpConfig/Extensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace workspacer.FSharpConfig
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class Extensions
+    {
+        public static Assembly LoadFile(this AppDomain appDomain, string filename) {
+            FileStream fs = new FileStream(filename, FileMode.Open);
+            byte[] buffer = new byte[(int) fs.Length];
+            fs.Read(buffer, 0, buffer.Length);
+            fs.Close();
+
+            return appDomain.Load(buffer);
+        }
+
+        public static AppDomain FileResolver(this AppDomain resoleAppDomain, IEnumerable<string> dllFiles)
```

Comment:
This is a method and should have a verb-y name, but FileResolver is a noun. If it's function is to idempotently load dll-files... Then why not simply call it `LoadFiles()`?

Also the extended AppDomain instance has a typo in it (resolveAppDomain), Since this is an extension method, I think a more appropriate name would simply be `instance`.